duckduckgo_search/ddg_images.py

```diff
@@ -138,7 +138,12 @@ def get_ddg_images_page(page):
 
     # download images
     if download:
-        keywords = keywords.replace('"', "'")
+        keywords = (
+            keywords.replace('"', "'")
+            .replace("site:", "")
+            .replace(" ", "_")
+            .replace("/", "_")
```

Comment:
This logic is repeated in multiple sections, which may lead to updates in 1 place but not the other. Would refactoring this logic into a single function allow for code re-use?